coordinator/txmanager.go

```diff
@@ -126,9 +129,37 @@ func (t *TxManager) syncSCVars(vars common.SCVariablesPtr) {
 
 // NewAuth generates a new auth object for an ethereum transaction
 func (t *TxManager) NewAuth(ctx context.Context, batchInfo *BatchInfo) (*bind.TransactOpts, error) {
-	gasPrice, err := t.ethClient.EthSuggestGasPrice(ctx)
-	if err != nil {
-		return nil, tracerr.Wrap(err)
+	//First we try getting the gas price from etherscan. If it fails we get the gas price from the ethereum node.
+	var gasPrice *big.Int
+	if t.etherscanService != nil {
+		etherscanGasPrice, err := (*t.etherscanService).GetGasPrice(ctx)
+		if err != nil {
+			log.Warn("Error getting the gas price from etherscan. Trying another method. Error: ", err.Error())
```

Comment:
Try to avoid long lines. You can break for a better clean code

Reply:
I agree. 
However, could we set to 160 columns?
80 columns were for old punch cards https://softwareengineering.stackexchange.com/questions/262723/why-were-punch-cards-used-for-programming  or for 80' or 90' ancient monitors.